### ISSUE TITLE
The 1.7 version is incompatible with IntelliJ IDEA 2021.3.3, IdeaVim 1.10.1, and AceJump 3.8.7 now

### ISSUE THREAD

**Issue by cychen2021** [AlexPl292/IdeaVim-EasyMotion#76]
I set
```vim
map <leader>s <Plug>(easymotion-s)
```
in my  `.ideavimrc`. After I press `<leader>s` in the normal mode, it behaves abnormally: for example, if the line is as follows
```java
class Main {
```
it will modify the line to
```java
classsymotion-s) class Main {
```
and switch to the insert mode.

**Comment by zhenchuanzhiaa** [comment]
I get it too ！

**Comment by AlexPl292** [comment]
Hi folks!
1) Do you have https://plugins.jetbrains.com/plugin/13360-ideavim-easymotion plugin installed?
2) Do you have `set easymotion` in your `.ideavimrc`?
3) Will it work with 1.9 version of IdeaVim-EasyMotion?

If everything above doesn't help, could you please send your `.ideavimrc` file and [logs](https://intellij-support.jetbrains.com/hc/en-us/articles/207241085-Locating-IDE-log-files)?

**Comment by cychen2021** [comment]
> Hi folks!
> 
> 1. Do you have https://plugins.jetbrains.com/plugin/13360-ideavim-easymotion plugin installed?
> 2. Do you have `set easymotion` in your `.ideavimrc`?
> 3. Will it work with 1.9 version of IdeaVim-EasyMotion?
> 
> If everything above doesn't help, could you please send your `.ideavimrc` file and [logs](https://intellij-support.jetbrains.com/hc/en-us/articles/207241085-Locating-IDE-log-files)?

It has been fixed by the 1.9 version.
